src/components/Login/Login.js

```diff
@@ -25,7 +30,78 @@ function Login() {
     setPass(e.target.value);
   };
   const submitHandler = () => {
-    console.log(username, pass);
+    setUsernam('');
+    setPass('');
+    if (logReg.status === false) {
+      fetch(
+        'https://identitytoolkit.googleapis.com/v1/accounts:signUp?key=AIzaSyBRegmu6l6G00JTkDtLcmhD97rD2NBg2cU',
```

Comment:
store this in some utils function called getEndpointsUrl () => someUrl: () => 'someurl'